Dockerfile

```diff
@@ -1,17 +1,25 @@
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ubuntu:24.04 AS base
+FROM ubuntu:20.04 AS base
```

Comment:
The `wait` command on an older version of bash used on Ubuntu 20.04 does not accept the `-p` argument. Please modify this line:

https://github.com/crystaldb/autodba/blob/13703811d28dd0f4e3450a0976dd44976c56e7bb/entrypoint.sh#L119

to:

```
wait -n # wait for any job to exit
EXITED_PID=$!
```